docs/add_custom_features.md

```diff
@@ -0,0 +1,125 @@
+# Add custom features for custom analysis
+
+If you want to perform custom analysis with your custom features that are not supported in the original task processors, and these features are only related to one particular system/dataset instead of the task itself, you can define the custom features and analysis in the `metadata` section in the output JSON file. 
+
+## Example of bucket analysis with discrete feature
```

Comment:
Sorry to suggest changes multiple times, but could we give an example that has discrete features and continuous features in the same section/JSON? It seems that splitting it into two sections is a bit repetitive.

Otherwise this looks great, and I'm happy to approve after this is resolved!